spec/controllers/api/v1/feedbacks_controller_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::FeedbacksController, type: :controller do
+    describe 'POST #create' do
+        let(:receiver) { create(:receiver) }
+        let(:giver) { receiver.giver }
+        let(:gift_found_feedback_params) do
+            { feedback: {
+                response: 'gift_found'
+            } }
+        end
+        let(:phone_number) { '+5691234567' }
+        let(:email) { 'example_email@example.com' }
+        let(:gift_not_found_feedback_params) do
+            { feedback: {
+                    response: 'gift_not_found',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.